AISKULight/rollup.config.js

```diff
@@ -23,7 +23,7 @@ updateDistEsmFiles(replaceValues, banner, true, true, "dist-es5");
 
 export default createConfig(banner, 
   {
-    namespace: "Microsoft.ApplicationInsights3",
+    namespace: "Microsoft.ApplicationInsights",
```

Comment:
Resetting all of the global namespaces back to the standard default "global" name as apart from the AISKU, none of the other browser bundles included the code to "assign" back to the global namespace (which was an unintended bug)